web/profiles/custom/os2loop/modules/os2loop_flag_content/README.md

```diff
@@ -2,4 +2,4 @@
 
 This module makes users able to write the editorial office regarding specific content.
 
-The config is defined in Flag Content Admin Form (/admin/config/os2loop_flag_content/settings).
+The config can be edited in admin configuration (/admin/config).
```

Comment:
This should be something along the lines of “Configuration » OS2Loop » OS2Loop Flag content settings (`/admin/config/os2loop/os2loop_flag_content/settings`)”.